### DISCUSSION TITLE
Help with installation

### DISCUSSION THREAD

**Discussion by Marcs1987** [FreshRSS/FreshRSS#5860] [Q&A] [answered]
So this is probably a real newbie question. I trying to install FreshRSS on AlmaLinux9. I've gotten to the step where it checks for directory access, "Installation · FreshRSS: step 2". All the PHP tests are green. But I get "Blast! Check permissions on the /usr/share/FreshRSS/data directory for apache user. The HTTP server must have write permission." Same for data/cache, data/users and data/favicons. Check for temp directory is green though. The rights for /usr/share/FreshRSS/data looks like
drwxrwxr-x.   9 apache apache

That should be all the rights needed?

**Comment by Alkarex** [comment]
I am not sure of the convention for your distribution, but try www-data for the group.
You can compare with what you might have in /var/www/html

**Reply by Marcs1987** [reply]
userId www-data doesn't exist on my system. There's userId apache that's basically the same

**Reply by Alkarex** [reply]
For the group

**Reply by Marcs1987** [reply]
Ah right. There isn't a group called www-data either, just apache

**Comment by Marcs1987** [comment]
Figured out it was a selinux issue and followed this page to resolve it
https://freshrss.github.io/FreshRSS/en/users/07_Frequently_Asked_Questions.html

Now the next problem. On page "Installation · FreshRSS: step 3" I'm asked to select database and presented with the option of either SQLite or MySQL/MariaDB. But I have neither of them installed, instead I have PostgreSQL installed, which I thought was the recommended one.

Why doesn't it show up as an option?

**Reply by Alkarex** [reply, marked as answer]
You need to have the corresponding PHP packages, such as `php-sqlite3 php-mysql php-pgsql` or the equivalent for your distribution.
I would recommend SQLite in your case, which does not require running a database service.

You could also consider our ready-to-use Docker image, which would avoid the troubles you are facing
https://hub.docker.com/r/freshrss/freshrss/
